### PR TITLE
creating a new Prime Server formula

### PR DESCRIPTION
**Pull request by drewda**
see https://github.com/kevinkreiser/prime_server

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

On the last item, I get the follow error:

```bash
Error: Error running `rubocop --format json --force-exclusion --parallel --except NewFormulaAudit --config /usr/local/Homebrew/Library/.rubocop_audit.yml /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/prime_server.rb`
```

I would appreciate any help others can give debugging that error.

-----

### CODE REVIEW

File: Formula/prime_server.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class PrimeServer < Formula
+  desc "non-blocking (web)server API for distributed computing and SOA based on zeromq"
+  homepage "https://github.com/kevinkreiser/prime_server"
+  head "https://github.com/kevinkreiser/prime_server.git", tag: "0.6.3"
```
Comment (woodruffw) [inline Formula/prime_server.rb]: Use `url` with a tagged release, not `head` with a git tag:

```ruby
url "https://github.com/kevinkreiser/prime_server/archive/0.6.3.tar.gz"
sha256 "<hash of file above>"
```

File: Formula/prime_server.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    pipe_output("prime_serverd").include?("Usage: prime_serverd num_requests|server_listen_endpoint concurrency")
```
Comment (woodruffw) [inline Formula/prime_server.rb]: New formulae in homebrew-core are required to have a functionality test, not just a `help`, `version`, or `usage` test. Check out [zeromq's test block](https://github.com/Homebrew/homebrew-core/blob/30abddeed8f2b8d6e3d7b5ee313ddcd2c8ff1671/Formula/zeromq.rb#L55) as an example of this.

**Review by woodruffw — CHANGES_REQUESTED** [review]

**Comment by woodruffw** [comment]
Thanks for contributing to Homebrew! I've left some review comments above.

**Comment by woodruffw** [comment]
Just realized this doesn't meet our notoriety requirements for inclusion in homebrew-core:

```
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

Please consider putting this in [a custom tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md) for the time being.

Sorry, and thanks!

**Comment by drewda** [comment]
@woodruffw O.K., thanks for the useful feedback and the prompt response!

**Pull request closed without merging**